lib/descriptor.js

```diff
@@ -27,61 +27,65 @@ const generateDescriptor = async (data, dpName = "scratchpad") => {
 }
 module.exports.generateDescriptor = generateDescriptor
 
-const generateDescriptorForLocalFiles = function (dpName = "scratchpad") { 
+const generateDescriptorForLocalFiles = (dpName = "scratchpad") => { 
   return new Promise(function(resolve, reject) { 
     fs.readdir('./', (err, files) => {
       var descriptor = {
```

Comment:
@zelima sorry about that, I am in progress of learning js, and this is my first javascript PR. Any comments will be appreciated